recipes/wordcloud/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.2.1" %}
+
+package:
+    name: wordcloud
+    version: {{ version }}
+
+source:
+    url: https://github.com/amueller/word_cloud/archive/{{ version }}.tar.gz
+    fn: wordcloud-{{ version }}.tar.gz
+    sha256: 936266403526943c8649bdc9bc6167e8c7eaba12e56f8ab286d1920617cef44e
+
+build:
+    number: 0
+    script:
+      # FIXME: This is a hack to make sure the environment is activated.
+      # The reason this is required is due to the conda-build issue
+      # mentioned below.
+      #
+      # https://github.com/conda/conda-build/issues/910
+      #
+      - source activate "${CONDA_DEFAULT_ENV}"  # [unix]
```

Comment:
Hah alright.  I've been checked-out from conda-forge for the last couple of months. I've missed so much!  Is the current thinking back to `python setup.py install` for distutils and add `[[-single-version-externally-managed --record=record.txt]]` for setuptools?